asyncscope.md

```diff
@@ -359,39 +370,35 @@ Examples of use
 
 ## Spawning work from within a task
 
-Use a `counting_scope` in combination with a `system_context` from [@P2079R2] to spawn work from within a task and join
-it later:
+Use a `let_with_async_scope` in combination with a `system_context` from [@P2079R2] to spawn work from within a task:
 ```cpp
 namespace ex = std::execution;
 
 int main() {
     ex::system_context ctx;
     int result = 0;
-    ex::counting_scope scope;
 
     ex::scheduler auto sch = ctx.scheduler();
 
-    ex::sender auto val = ex::on(sch, ex::just() | ex::then([&result, sch, &scope]() {
+    ex::sender auto val = ex::on(sch, ex::just() | ex::let_with_async_scope([sch](auto scope) {
```

Comment:
I've been imagining `let_with_async_scope` as taking a callable and no predecessor, in which case this needs to remove `just()`.
```suggestion
    ex::sender auto val = ex::on(sch, ex::let_with_async_scope([sch](auto scope) {
```